lib/galaxy/tool_util/verify/asserts/text.py

```diff
@@ -20,6 +20,13 @@ def assert_has_line(output, line):
     assert match is not None, "No line of output file was '%s' (output was '%s') " % (line, output)
 
 
+def assert_has_n_lines(output, n):
+    """ Asserts the specified output if the number of lines match the given number."""
+    for counter, line in enumerate(output.split('\n')):
```

Comment:
```suggestion
    n_lines_found = len(output.splitlines())
```
will also work for other types of newlines, if we don't want that we can also use `output.count('\n')`.

Reply:
I was just concerned about memory usage, but we assume that test data stays small?